Source/WebCore/loader/ApplicationManifestLoader.h

```diff
@@ -40,7 +41,7 @@ class CachedApplicationManifest;
 class DocumentLoader;
 
 class ApplicationManifestLoader final : private CachedRawResourceClient {
-WTF_MAKE_NONCOPYABLE(ApplicationManifestLoader); WTF_MAKE_FAST_ALLOCATED;
+WTF_MAKE_NONCOPYABLE(ApplicationManifestLoader); WTF_MAKE_FAST_ALLOCATED_WITH_HEAP_IDENTIFIER(Loader);
```

Comment:
Nit: This should be two lines.

Reply:
This style is also the cause of the `style` bot failures.

Reply:
Yeah as you suspected I just kept the style that was there before.

Reply:
Interestingly it's not the only line like this but the style checker only caught one. And it caught a different one on my local machine!